src/upload/application/views/home/index_body.php

```diff
@@ -194,14 +194,8 @@
             <td style="width:18px; background: url('{img_path}dropdownmenu_arrow.png') no-repeat scroll 0 0 #8D9AA7;"></td>
         </tr>
         </tbody></table>
-        <input class="js_uniUrl" type="hidden" name="uni_url" id="uni_domain" value="">
+        <input class="js_uniUrl" type="hidden" name="uni_url" id="uni_domain" value="{base_path}">
         <div id="uni_selection" style="display: none;">
-                                    <script type="text/javascript">
-                        <!--
-                            select_uni('{base_path}'.replace('http://', '').replace('https://', '').slice(0, -1), '','');
-                        //-->
-                        </script>
-
```

Comment:
@duhow this should not be part of this PR, it is unrelated

Reply:
No, this is required. If "universe" is not set or empty, the JS function will **POST** to another domain.
Also as the Javascript code uses the full URL and not the `base_path`, having this code just breaks the domain.
So while we don't allow multi-universe, this code can be removed.